ts/features/idpay/configuration/store/actions/index.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  ActionType,
+  createStandardAction,
+  createAsyncAction
+} from "typesafe-actions";
+import { InstrumentListDTO } from "../../../../../../definitions/idpay/InstrumentListDTO";
+import { NetworkError } from "../../../../../utils/errors";
+
+export type IdPayInitiativeInstrumentsGetPayloadType = {
+  initiativeId: string;
+  isRefreshing?: boolean;
+};
+
+export const idpayInitiativeInstrumentsGet = createAsyncAction(
+  "IDPAY_INITIATIVE_INSTRUMENTS_GET_REQUEST",
+  "IDPAY_INITIATIVE_INSTRUMENTS_GET_SUCCESS",
+  "IDPAY_INITIATIVE_INSTRUMENTS_GET_FAILURE"
+)<IdPayInitiativeInstrumentsGetPayloadType, InstrumentListDTO, NetworkError>();
+
+export type IdPayInitiativeInstrumentsDeletePayloadType = {
+  initiativeId: string;
+  instrumentId: string;
+};
+
+export type IdPayInitiativeInstrumentsDeleteErrorPayloadType = {
+  instrumentId: string;
+  error: NetworkError;
+};
+
+export const idpayInitiativeInstrumentDelete = createAsyncAction(
+  "IDPAY_INITIATIVE_INSTRUMENT_DELETE_REQUEST",
+  "IDPAY_INITIATIVE_INSTRUMENT_DELETE_SUCCESS",
+  "IDPAY_INITIATIVE_INSTRUMENT_DELETE_FAILURE"
+)<
+  IdPayInitiativeInstrumentsDeletePayloadType,
+  IdPayInitiativeInstrumentsDeletePayloadType,
+  IdPayInitiativeInstrumentsDeleteErrorPayloadType
+>();
+
+export const idPayInitiativesInstrumentRefreshStart = createStandardAction(
+  "IDPAY_INITIATIVES_INSTRUMENT_REFRESH_START"
+)<IdPayInitiativeInstrumentsGetPayloadType>();
+
+export const idPayInitiativesInstrumentRefreshStop = createStandardAction(
+  "IDPAY_INITIATIVES_INSTRUMENT_REFRESH_STOP"
+)();
```

Comment:
We are refreshing the initiative instruments, and not the opposite. Actions name should describe what the action does.

```suggestion
export const idPayInitiativeInstrumentsRefreshStart = createStandardAction(
  "IDPAY_INITIATIVE_INSTRUMENTS_REFRESH_START"
)<IdPayInitiativeInstrumentsGetPayloadType>();

export const idPayInitiativeInstrumentsRefreshStop = createStandardAction(
  "IDPAY_INITIATIVE_INSTRUMENTS_REFRESH_STOP"
)();
```

Reply:
Addressed into 88119f5289e559fb5bbe5b9e3900263d585df0ce